wcivf/apps/people/templates/people/includes/_person_previous_elections_card.html

```diff
@@ -0,0 +1,34 @@
+{% load humanize %}
+
+{% if object.past_not_current_candidacies %}
+<section class="ds-card">
+<div class="ds-table">
+    <table>
+    <caption style="font-style: normal;">
+        <h3 class="ds-padded">Previous Elections</h3>
+    </caption>
+    <tr>
+        <th>Date</th>
+        <th>Election</th>
+        <th>Party</th>
+        <th>Results</th>
+    </tr>
+    <tr>
+        {% for person_post in object.past_not_current_candidacies %}
+        {{ person_post.post_election.short_cancelled_message_html }}
+        {{ person_post.post_election.short_cancelled_message_html }}
+        {{ person_post.post_election.short_cancelled_message_html }}
+        <td>{{ person_post.election.election_date|date:"Y" }}</td>
+        <td>{{ person_post.post.label }}: {{ person_post.election }}</td>
+        <td>{{ person_post.party.party_name }}</td>
+        {% if person_post.elected %}
+            <td>{% if person_post.votes_cast %}{{ person_post.votes_cast|intcomma }} votes (elected){% else %}Vote count not available{% endif %}</td>
```

Comment:
So I think in the `else` here, we would want to say "Elected" rather than "Vote count not available"? Because we know they were elected (from the check on line 24) we just don't know how many votes they got

Reply:
Yes, now that I think about it, maybe we want to say both where the vote count is not available.